res/client/client.css

```
@@ -986,6 +993,11 @@ QPushButton#searchButton, #advSearchButton, #fafDbButton, #spookyDbButton
     background-color: rgb(39, 39, 45);
 }
 
+QPushButton#searchButton::hover, #advSearchButton::hover, #fafDbButton::hover, #spookyDbButton::hover
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: hover

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)